packages/flutter/lib/src/widgets/router.dart

```diff
@@ -976,8 +976,8 @@ class ChildBackButtonDispatcher extends BackButtonDispatcher {
 ///
 /// This delegate is used when the [Router] widget is first built with initial
 /// route information from [Router.routeInformationProvider] and any subsequent
-/// new route notifications from it. The [parseRouteInformation] widget calls
-/// the [parseRouteInformation] with the route information.
+/// new route notifications from it. The [Router] widget calls the
+/// [parseRouteInformation] with the route information from [routeInformationProvider].
```

Comment:
This should be [Router.routeInformationProvider]

Reply:
fixed :)